tests/test_lightblue.py

```diff
@@ -1306,9 +1310,77 @@ def test_find_all_repositories(self, exists, gql_client):
             self.fake_repositories_with_content_sets}
         self.assertEqual(ret, expected_ret)
 
+    @mock.patch('freshmaker.pyxis_gql.Client')
     @patch('freshmaker.lightblue.LightBlue.find_container_images')
     @patch('os.path.exists')
-    def test_find_images_with_included_srpm(self, exists, cont_images):
+    def test_find_images_with_included_srpm(self, exists, cont_images, mocked_client):
+        mocked_client.return_value.execute.return_value = {
+        "find_images": {
+            "data": [
+                {
+                    "architecture": "amd64",
+                    "brew": {"build": "parent-1-2"},
+                    "content_sets": ["dummy-content-set-1"],
+                    "edges": {
+                        "rpm_manifest": {
+                            "data": {
+                                "rpms": [
+                                    {
+                                        "name": "openssl",
+                                        "nvra": "openssl-1.2.3-2.module+el8.0.0+3248+9d514f3b",
```

Comment:
This is not a valid `nvra`, it doesn't include `arch`, it will cause problem in code to parse this value.